src/rtt/rttinlin.c

```diff
@@ -1567,6 +1552,183 @@ struct node *n;
    fname = cname;
    line = 7;
 
+#ifndef OLD_SCHOOL
```

Comment:
I suggest we use `LEGACY_FUNC_PROTOTYPE` , just to make it clearer, instead of `OLD_SCHOOL`.